lib/logstash/outputs/sumologic/common.rb

```diff
@@ -69,5 +69,15 @@ def log_dbg(message, *opts)
       end
     end # def log_dbg
 
+    def blank?(value)
+      if value.kind_of?(NilClass)
+        true
+      elsif value.kind_of?(String)
+        value !~ /\S/
```

Comment:
Is it faster than `value.strip().length == 0`? Just curious

Reply:
I didn't go into details, just copied over from LogStash original method https://github.com/elastic/logstash/blob/d176e608bdd20a3221bdbb8b3fc84fc3f1085229/logstash-core/lib/logstash/util.rb#L228